agences-regionales-sante/auvergne-rhone-alpes/2020-03-13.yml

```diff
@@ -1,78 +0,0 @@
-date: 2020-03-12
```

Comment:
I don't think this removal is related to that pull-request.

Reply:
This branch is not up to date.